{
  "comments": [
    {
      "key": {
        "uuid": "2779197d_0c3e14ff",
        "filename": "src/config/device-manager/device_manager/dm_utils.py",
        "patchSetId": 1
      },
      "lineNbr": 405,
      "author": {
        "id": 843
      },
      "writtenOn": "2020-04-29T13:36:29Z",
      "side": 1,
      "message": "can you add default value for router_external\u003dFalse, so there will not be any impact.",
      "revId": "ccf7881880532dee790686fa1f2f13f98faf8a78",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    }
  ]
}